cico_functions.sh

```diff
@@ -40,9 +40,9 @@ function install_deps() {
   /usr/sbin/setenforce 0  || true
 
   # Get all the deps in
-  yum install -y yum-utils device-mapper-persistent-data lvm2
+  yum install -d1 -y yum-utils device-mapper-persistent-data lvm2
```

Comment:
since yum is now dnf, maybe using `-q` is better than deprecated flag `-d` ?

Reply:
The issue is that `-q` is not equivalent to `-d1` -- `yum install -q` does not list the packages that were installed.

If I remember correctly, `dnf` is not available on the CI servers.